modules/nodes-nodes-swap-memory.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+[id="nodes-nodes-swap-memory_{context}"]
+
+= Managing swap memory on nodes
+
+:FeatureName: Configuring swap memory usage on nodes
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+In {product-title}, you can configure nodes to use swap memory. By default, nodes will not start in the presence of swap memory.
```

Comment:
| By default, nodes will not start in the presence of swap memory.
What does this mean?

Reply:
Do we need to explain what swap memory is?